README.md

```diff
@@ -69,8 +64,11 @@ cab be found here:
 
 ## Comprehensive user guide
 
-See [HDL User Guide](https://wiki.analog.com/resources/fpga/docs/hdl) for a more
-detailed guide.
+See [HDL User Guide](https://wiki.analog.com/resources/fpga/docs/hdl) for a more detailed guide.
+
+## Documentation
```

Comment:
Documentation is part of the Comprehensive user guide. I don't think we need a link to the resources/fpga as the projects on that page are mostly outdated, or at least we don't have support for them in this repository.

Reply:
OK, will be removed.